crates/router/src/core/admin.rs

```diff
@@ -150,6 +156,16 @@ pub async fn create_merchant_account(
                 })
         })
         .transpose()?;
+    let fingerprint = Some(create_merchant_fingerprint());
+    if let Some(fingerprint) = fingerprint {
+        db.insert_config(configs::ConfigNew {
+            key: format!("secret_{}", req.merchant_id),
+            config: fingerprint,
+        })
```

Comment:
Call it `fingerprint_secret`. Just fingerprint or just secret is generic and misleading.